### PR TITLE
Add Range exponential combinators

### PR DESCRIPTION
**Pull request by moodmosaic**
Implementation + doctests are based on the [current (v3.0) Haskell version](https://github.com/hedgehogqa/haskell-hedgehog/blob/0.3/hedgehog/src/Hedgehog/Range.hs).

### CODE REVIEW

File: src/Hedgehog/Range.fs [hunk 1]
```diff
@@ -164,6 +164,22 @@ module Range =
 
             fromBigInt (z + diff)
 
+        /// Scale a floating-point number exponentially with the size parameter.
```
Comment (moodmosaic) [inline src/Hedgehog/Range.fs]: Oh, this should rather be

```
Scale an integral exponentially with the size parameter.
```

Fixing it 😄

Comment (moodmosaic) [inline src/Hedgehog/Range.fs]: Done

**Comment by moodmosaic** [comment]
Knock knock, anyone there? 🤠

**Comment by jystic** [comment]
🎉

**Pull request merged**